src/components/facts/facts.js

```diff
@@ -0,0 +1,189 @@
+import React, { PropTypes, Component } from 'react';
+import ImmutablePropTypes from 'react-immutable-proptypes';
+import moment from 'moment';
+import { connect } from 'react-redux';
+
+import { contextTypes } from '../../utils/decorators/withContext.js';
+import withStyles from '../../utils/decorators/withStyles.js';
+import styles from './Facts.css';
+import withViewport from '../../utils/decorators/withViewport.js';
+import Accordion from 'react-bootstrap/lib/Accordion';
+import Panel from 'react-bootstrap/lib/Panel';
+
+
+@withViewport
+@withStyles(styles)
+export default class Facts extends Component {
+  static contextTypes = contextTypes;
+
+  render() {
```

Comment:
Volgende foutmeldingen verschijnen in de console:

´´´
Warning: validateDOMNesting(...): <div> cannot appear as a descendant of <p>. See Facts > p > ... > div.warning @ (program):45
(program):45 Warning: validateDOMNesting(...): <figure> cannot appear as a descendant of <p>. See Facts > p > ... > figure.warning @ (program):45
(program):45 Warning: validateDOMNesting(...): <figcaption> cannot appear as a descendant of <p>. See Facts > p > ... > figcaption. 
´´´